src/components/controller/ContactNavigation/index.js

```diff
@@ -0,0 +1,27 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { logoutUserRequest, toggleEventDetail } from 'actions';
```

Comment:
`toggleEventDetail` is imported but not used

Reply:
actually now that I look at it, I don't even see toggleEventDetail defined in `src/actions/index.js`